examples/docker/pub/main.py

```diff
@@ -0,0 +1,44 @@
+# BSD 3-Clause License
+# Stef van der Struijk
+
+import argparse
+import zmq
+import time
+
+
+def publisher(ip="0.0.0.0", port=5551):
+    # ZMQ connection
+    url = "tcp://{}:{}".format(ip, port)
+    print("Going to connect to: {}".format(url))
+    ctx = zmq.Context()
+    socket = ctx.socket(zmq.PUB)
+    socket.connect(url)  # publisher connects to subscriber
```

Comment:
If this is a publisher, shouldn't this be a `socket.bind` call?

Reply:
Thank you for checking @JSalazar88. I agree that it's more common that a single Publisher sends message to multiple Subscribers, but with ZMQ it is also possible to have a single Listener wait for messages from multiple Publishers.
In the example code, there is only 1 Pub and 1 Sub, so it doesn't really matter which socket connects and which one binds. This example is now setup as a m-Pub to 1-Sub pattern. Could be changed, but it won't change anything for the functioning of this example.